plugin/pom.xml

```diff
@@ -109,6 +109,16 @@
       <artifactId>fabric8-maven-generator-webapp</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
It would be super awesome if we could get rid of the [spring-boot dependency](https://github.com/nicolaferraro/fabric8-maven-plugin/blob/3a9ff06301becc26d4f3510a67896f219ab73a04/plugin/pom.xml#L32-L42) in this pom.xml and only have it in the watcher module for now. This plugin pom.xml ideally should be completely agnostic to the specific technology the builds are using (vert.x, spring-boot, wildlfy-swarm, ....)